neqo-transport/src/ackrate.rs

```diff
@@ -43,7 +43,7 @@ impl AckRate {
 
     pub fn write_frame(&self, builder: &mut PacketBuilder, seqno: u64) -> bool {
         builder.write_varint_frame(&[
-            FRAME_TYPE_ACK_FREQUENCY,
+            FrameType::AckFrequency.into(),
```

Comment:
```suggestion
            u64::from(FrameType::AckFrequency),
```

Just a preference for not hiding types.